assets/javascripts/vendor/awin-11152.js

```diff
@@ -0,0 +1 @@
|isObservedInput|regPattern|attachOnChangeInput|autoCompleteChecker|continue|sVarName|callback|_aw_xid|jp|encode|awcRegex|xid|sEncodedString|sValue|sExpires|_getCookieDomain|_getAWaidValue|emailPattern|isEmailAddress|fromIndex|twoPartsTldDomains|_getDomain|is|twoPartTld|abs|tldDomains|tld|searchElement|while|throw|prototype|anchorAtp|hasInputBeenObserved|salt|cookieDomain|pixelCall|pixel|CrossDeviceTracking|openIframe|attachEvent|emailHash|cookie|channel|th|d9Data|readyState|T2|biz|iScriptCount|getScriptAppendNode|aRequiredVars|BasketImage|org|sUrl|aScripts_loop|sLinePiece|no|javascript|device9Url|Sigma1|Sigma0|sigma0|za|sFrameSrc|sigma1|encodeURIComponent|kr|nz|Maj|sImageSrc|zw|zm|vn|ye|yu|ws|vu|ua|tz|hasOwnProperty|ve|parentNode|vg|insertBefore||va|uz|uy||ug|match|digestClickId|D9v|buildQueryString|d9tag|async|vi|device9|pe|gh|gg|gf|ge|gl|gm|gq|gp|gn|gd|ga|eu|er|eg|ee|fi|fj|fo|fm|fk|gr|gs|jm|je|ir|io|jo|ke|ki|kh|kg|im|ie|hk|gy|gu|gt|hm|hn|hu|ht|hr|ec|dz|mil|es|se|nl|gw|wf|mobi|sj|yt|it|ru|d9core|fp|http|protocol|edu|gov|us|fr|de|eh|mh|pro|museum|jobs|info|travel|dj|do|dm|dk|coop|aero|um|iq|kp|cat|arpa|pm|so|td|gb|km|kn|ro|re|qa|py|rw|sa|sd|sc|sb|pw|pt|pg|pf|pop|pa|ph|pk|ps|pr|pn|sg|sh|tl|tk|tj|tg|tm|tn|tr|tp|to|tf|tc|sm|sl|sk|si|sn|sr|sz|sy|st|om|nu|ma|ly|lv|lu|mc|md|ml|mk|mg|lt|ls|la|kz|ky|kw|lb|lc|lr|lk|li|mm|mn|ne|nc|na|mz|nf|ng|nr|np|ni|my|mx|mr|mq|mp|mo|ms|mt|mw|mv|mu|tw|sku|0xd6990624|0xf40e3585|0x106aa070|0x19a4c116|0xd192e819||0xc76c51a3|0x92722c85|0xa2bfe8a1|0xa81a664b|0xc24b8b70|0x1e376c08|0x2748774c|0x748f82ee|0x78a5636f|0x84c87814|0x8cc70208|0x682e6ff3|0x5b9cca4f|0x34b0bcb5|0x391c0cb3|0x4ed8aa4a|0x81c2c92e|0x766a0abb|0x5cb0a9dc|0x76f988da|0x983e5152|0xa831c66d|0x4a7484aa|0x2de92c6f|0xe49b69c1|0xefbe4786|0x0fc19dc6|0x240ca1cc|0xb00327c8|0xbf597fc7|0x2e1b2138|0x4d2c6dfc|0x53380d13|0x650a7354|0x27b70a85|0x14292967|0xc6e00bf3|0xd5a79147|0x06ca6351|0x90befffa|0xa4506ceb|2000|apply|INPUT|QX4QkKEU|setTimeout|0x1f|u00ef|0x0f|u00c0|u00df|about|blank|Infinity|max|awin_tntc|yes|Object|defined|TypeError|or|not|u00e0||||||decode||0xa54ff53a|0x510e527f|||||||||||0x9b05688c|||0x1f83d9ab||0x3c6ef372|z_|0xbef9a3f7|0xc67178f2|0x6a09e667|0x5be0cd19|ceil|u0800|uffff|0xe0|0x3F|0xc0|u07ff|pow|floor|0xf|0xc19bf174|0xbb67ae85|0x9bdc06a7|OrderID|OrderTotal|SiteID|AdvID|get|Error|oError|ia|255|basket|navigator|appName|Microsoft|Internet|category|quantity|product_line|||getBasketData||||||||price|loaded||||complete|365||||1000|||expires|||toGMTString|getTime|setTime|Z_|parseanchorAtp|Date|path|gclid_|img|_aw_script_|onreadystatechange|pixelAppend|frameAppend|getAffiliateId|head|html|Explorer|1062|0x428a2f98|err|getAttribute|confirm|0x71374491|0xe9b5dba5|0xab1c5ed5|0x923f82a4|0x59f111f1|0x3956c25b|content|XPathResult|CCampID|11152|ImpID|gv|3055|CampID||ANY_TYPE|||iterateNext|||||node||||||selectNodes|0xd807aa98|||0xb5c0fbcf|||fetchZxParam||||instanceof|innerHTML|cks|0x72be5d74|META|zx|href|0x243185be|0x12835b01|0x80deb1fe|0x550c7dc3|pv|parts'.split('|'),0,{}))
```

Comment:
`eval` 😱 I really don't trust this script!

Reply:
It/Awin has been audited by the Infosec team